### ISSUE TITLE
Add Tags/genre

### ISSUE THREAD

**Issue by Nabeelshar** [nanihadesuka/NovelDokusha#21]
Add tags system when can be shown below description and using the same code can be added author alt title and more.

**Comment by nanihadesuka** [comment]
That information would be found on the "databases" page of that novel, which is more probable to have more information about the novel and (important) is consistent. For these feature, how would you design theses tags when every source can have different tags, authors or none at all?

**Comment by Nabeelshar** [comment]
Let me see if I can make a concept design.

**Comment by Nabeelshar** [comment]
Here it is and for the data every website has one genre at least.
https://balsamiq.cloud/s5r0rhi/peeadxs

**Comment by nanihadesuka** [comment]
Yes, we of course can add that for each source but that means having to maintain it for the `n` sources the app has. The benefit of only having those two databases that specialize in the metadata of the novels is that the app will get consistent info about a much larger catalogue and all the authors and related novels. This is for example one thing I would love to have when I use Tachiyomi but that it doesn't have, one of the reason I made the app this way :)

So the thing is, if you want to search by filters, genres, sort or whatever use the databases then, once you found what you want to read, use the global search to read from the source you like most.

**Comment by AhmadNaruto** [comment]
aah yes, I can accept your concept. maybe I'll focus more on creating a database. I'm trying it now. 
but there is one problem, because in our database we can't directly enter it into the library without involving the source. so is it okay to make one website into two events (source and database) 
other than that maybe the advantage of using the pafa source filter can make searching (not by title) easier. because it can be filtered by order and genre. I think so.

**Comment by nanihadesuka** [comment]
That thing you mention already happens, Novel Updates is a Database and also a Source. But just to be clear, I don't have any plans to change the way the app is structured, I'm open to extension but not modification.

**Comment by Nabeelshar** [comment]
Hmm, what about just rating? ⭐⭐⭐⭐⭐.
Why I want tags/genre was because I want know how good a novel is(novelupdates rating is not good, they rate novels 3/5 but novels are amazing) .
For example is mtlnovels as they are mtl novels they don't have an existence on any other sources except mtlnovels.

**Comment by nanihadesuka** [comment]
I don't think you are describing what you want clearly. You want to see a rating? where? is the rating something you grab from the databases (novel updates or baka-updates) or is only to be set inside the app locally. Is it really worth it to have all that tag and rating info when reading a novel, will it clutter the interface from the important core info the app wants to display? It it really worth it?

On the novel updates rating, can't comment how good it is, but its a statistic doesn't mean it must match your preferences.

**Comment by Nabeelshar** [comment]
![IMG_20220515_142635.jpg](https://user-images.githubusercontent.com/96086329/168466091-4055f0fb-32f9-4faf-8103-7579c088a7c6.jpg)

And data can be taken from novelinfo page.
